build/Gruntfile.coffee

```diff
@@ -77,6 +80,11 @@ module.exports = (grunt) ->
       dest: appDir
       ext: '.js'
 
+  babelConfig =
+    options: {}
```

Comment:
Ok, so I just grabbed the data by requiring `src/babel`, which is a bit weird. Do you think I should extract those options into some separate file, or just leave it like that for now?